descriptors/ModuleDescriptor-template.json

```diff
@@ -200,7 +200,7 @@
     },
     {
       "id": "orders-storage.receiving-history",
-      "version": "3.2",
+      "version": "4.2",
```

Comment:
should be 4.0

Reply:
major version in pieces interface also should be increased, because we renamed the field

Reply:
Done